jsonnet/kube-thanos/kube-thanos-store-default-params.libsonnet

```diff
@@ -12,7 +12,10 @@
   ignoreDeletionMarksDelay: '24h',
   logLevel: 'info',
   logFormat: 'logfmt',
+  maxTime: null,
+  minTime: null,
   resources: {},
+  storeGRPCSeriesSampleLimit: null,
```

Comment:
Putting nulls here has a few benefits IMO:

- it doesn't duplicate upstream defaults
- it documents the existence of these keys, and how they're formulated compared to the corresponding upstream flags (e.g. camelCase vs kebab-case)
- it assists the implementation of `util.withOptionalArgs` (see thread below)